src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public abstract class Machine {
+    protected String name = "Machine";
```

Comment:
if you want to have the field `name` let's remove initialization.
```
protected String name;
```

Reply:
removed